ApiEverest/CrudApi/Services/CustomerService.cs

```diff
@@ -0,0 +1,74 @@
+﻿using ApiEverest.Entities;
+
+namespace CustomerApi.Services
+{
+    public class CustomerService : ICustomerService
+    {
+        private readonly List<CustomerEntity> listCustomers = new List<CustomerEntity>();
+
+        public bool Create(CustomerEntity model)
+        {
+            model.Id = listCustomers.LastOrDefault()?.Id + 1 ?? 1;
+
+            if(!listCustomers.Any())
+            {
+                listCustomers.Add(model);
+                return true;
+            }
+            if(!customerDuplicate(model))
+            {
+                listCustomers.Add(model);
+                return true;
+            }
+            
+            return false;
+        }
```

Comment:
```suggestion
        public void Create(CustomerEntity model)
        {
            model.Id = listCustomers.LastOrDefault()?.Id + 1 ?? 1;

            if(listCustomers.Any() || customerDuplicate(model))
            {
                throw new ArgumentException();
            }

            listCustomers.Add(model);
        }
```
Ao invés de você fazer esses métodos retornando um booleano pra fazer a verificação se saiu tudo corretamente o ideal é não retornar nada e tratar isso com as exceções a aqui e resgatar elas na controller com o try catch